README.md

```diff
@@ -94,6 +94,17 @@ Test against a specific Puppet or Facter version:
     $ PUPPET_VERSION=2.7.19 bundle update && bundle exec rake spec
     $ FACTER_VERSION=1.6.8  bundle update && bundle exec rake spec
 
+## Puppet 3 support
+
+Puppet 3 is EOL as-of January 2017. The last release of this module that will
+support Puppet 3.X and earlier will be 1.2.5
```

Comment:
Suggest removing the exact version and just say that the v1 series will support Puppet3 and v2 will support Puppet4.